runtime/compiler/optimizer/J9RecognizedCallTransformer.cpp

```diff
@@ -106,7 +108,46 @@ void J9::RecognizedCallTransformer::transform(TR::TreeTop* treetop)
       case TR::java_lang_Math_min_L:
          processIntrinsicFunction(treetop, node, TR::lmin);
          break;
+      case TR::java_lang_StringUTF16_toBytes:
+         process_java_lang_StringUTF16_toBytes(treetop, node);
+         break;
       default:
          break;
       }
    }
+
+void J9::RecognizedCallTransformer::process_java_lang_StringUTF16_toBytes(TR::TreeTop* treetop, TR::Node* node)
```

Comment:
Could you move this function up so that all other process* methods are located together in the file?

Reply:
Fixed in b1e234c